guardian/models/__init__.py

```diff
@@ -0,0 +1,17 @@
+from .models import (
+    UserObjectPermissionBase,
+    UserObjectPermissionAbstract,
+    GroupObjectPermissionBase,
+    GroupObjectPermissionAbstract,
+    Permission,
+    Group
+)
+
+# Must import after .models
+# When .models is loaded, default generic object permissions are created
+# The following statements may redirect external references to custom
+# generic object permission models
+from guardian.utils import get_user_obj_perms_model, get_group_obj_perms_model
+UserObjectPermission = get_user_obj_perms_model()
+GroupObjectPermission = get_group_obj_perms_model()
+
```

Comment:
Please add `__all__ = […]` to silence unused-import warnings.

Reply:
added in 72c71f05caae667de69a66925608e140fcbe34fc

Also, note Permission and Group are included only because some files within guardian were importing these from guardian.models rather than from django.contrib.auth.models.  They have been left in just in case some downstream library was doing the same.